### PR TITLE
CircleCI: added QA automation

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -463,7 +495,8 @@ workflows:
           <<: *after-linter
       - valgrind:
           test_args: CLUSTER=0 AOF=0
-          <<: *on-any-branch
+          #@@ <<: *on-any-branch
+          <<: *never
```
Comment (alonre24) [inline .circleci/config.yml:499]: Why disabling valgrind?

Comment (chayim) [reply .circleci/config.yml:499]: Beat me to it. I'd argue that we should run valgrind on integration attempts not all branches -IMHO.

Comment (rafie) [reply .circleci/config.yml:499]: `#@@` is disabling for the sake of the PR, it takes about an hour to run. Re-enabling it once everything passes.
Basically, running valgrind here is Dvir's call, I prefer to either invoke it manually or nightly or when a PR is merged.
We can also try to enable CLang Sanitizers on RedisAI.

File: tests/qa/nightly.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "service_id": "single_module_test_cycle",
+  "name": "redisai automation-testing",
+  "properties": {
+    "sut_version": "master",
+    "email_recipients": "redisaidev-aaaacob2o7eeecrhkqwua77kku@redislabs.slack.com",
```
Comment (DvirDukhan) [inline tests/qa/nightly.json:6]: this is our slack channel?

File: tests/qa/release.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        "teardown": true,
+        "name": "bionic-amd64-aws",
+        "concurrency": 1,
+		"module_url": "http://redismodules.s3.amazonaws.com/{{RS_MODULE_FILE_PREFIX}}/{{RS_MODULE_FILE_PREFIX}}-cpu.linux-bionic-x64.{{MODULE_VERSION}}.zip"
+      },
+      {
+        "teardown": true,
+        "name": "rhel7.7-x86_64-aws",
+        "concurrency": 1,
+		"module_url": "http://redismodules.s3.amazonaws.com/{{RS_MODULE_FILE_PREFIX}}/{{RS_MODULE_FILE_PREFIX}}-cpu.linux-centos7-x64.{{MODULE_VERSION}}.zip"
```
Comment (DvirDukhan) [inline tests/qa/release.json:30]: will it run?

Comment (rafie) [reply tests/qa/release.json:30]: Right. I'll disable this one.

File: .circleci/config.yml [hunk 4]
```diff
@@ -167,7 +167,7 @@ jobs:
 
   build-and-test:
     docker:
-      - image: redisfab/rmbuilder:6.0.9-x64-buster
+      - image: redisfab/rmbuilder:6.2.1-x64-buster
```
[the remark this answers is not in the document]
Comment (rafie) [reply .circleci/config.yml:170]: We're working on a `config.yml` abstraction, but the build logic sets up it's own redis version anyway. I'm considering removing the redis version spec from `rmbuilder` altogether.

File: tests/qa/nightly.json (hunk 2 above)
Comment (rafie) [reply tests/qa/nightly.json:6]: It is. You can generate an email address from Slack: https://slack.com/intl/en-il/help/articles/206819278-Send-emails-to-Slack#:~:text=Click%20the%20details%20icon%20in,email%20address%20for%20the%20conversation.

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**